packages/editor-ui/src/modules/settings.ts

```diff
@@ -69,6 +70,17 @@ const module: Module<ISettingsState, IRootState> = {
 
 			context.commit('setPersonalizationAnswers', results);
 		},
+		async getPromptsData(context: ActionContext<ISettingsState, IRootState>) {
```

Comment:
@mutdmour The logic is little bit changed. We will use /prompts endpoint for both features, valueSurvey and contactPrompt.